commit e090681475db286333839603fb3668f3cdcfd3d1
Author: Hanno Schlichting <hanno@hannosch.eu>
Date:   Thu Feb 2 21:17:38 2012 +0100

    Fix for #33 - unwanted installation of data files.
    The readme and history files are part of the source dist thx to statements in the MANIFEST.in. This reverts `d6214ababb34f9fe5ae77fabb415ca87f0e3dd72` introduced in https://github.com/kennethreitz/clint/pull/22.

diff --git a/setup.py b/setup.py
index 78d9904..fd62e29 100755
--- a/setup.py
+++ b/setup.py
@@ -29,10 +29,6 @@ setup(
     author='Kenneth Reitz',
     author_email='me@kennethreitz.com',
     url='https://github.com/kennethreitz/clint',
-    data_files=[
-        'README.rst',
-        'HISTORY.rst',
-    ],
     packages= [
         'clint',
         'clint.textui',
